iOS_Sanity.py

```diff
@@ -0,0 +1,235 @@
+from appium import webdriver
+import unittest
+import time
+import requests
+import json
+
+
+class TestCases(unittest.TestCase):
+    # vars
+    myAddress = ''
+    myBalance = ''
+    badAddress = ''
+    noTrustAddress = 'GANFGTTCZL3D477BSCPR4RMUCX6RLERFUMOKZQYWK22ZFECZ3C7WXIZK'
+    qaAccount = 'GBTVB43S7PX2EZKXTHEXAGLVTRBXDOZY2V3LOYN2PSWRIOAFPRMUO2WA'
+
+    # Set up Appium and the app
+    @classmethod
+    def setUpClass(cls):
+        # Sample App should be already installed on the emulator # TODO: add option to install
+        bundleId = 'com.kinfoundation.KinSampleApp'
+        cls.driver = webdriver.Remote(
+            command_executor='http://127.0.0.1:4723/wd/hub',  # Run on local server
+            desired_capabilities={
+                'bundleId': bundleId,
+                'platformName': 'iOS',
+                'platformVersion': '11.2',
+                'deviceName': 'iPhone 7'  # TODO: name and version from somewhere else
+            }
+        )
+        cls._values = []
+        # Timeout for element searching
+        # Official documentation says its in millisecond, but it actually works in seconds for me
+        cls.driver.implicitly_wait(10)
+
+    # Called when the test run is over
+    @classmethod
+    def tearDownClass(cls):
+        # Re-open the app
+        cls.driver.reset()
+        # Enter the testnet account screen
+        testNetButton = cls.driver.find_element_by_accessibility_id('TestNetButton')
+        testNetButton.click()
+        # Delete your account
+        deleteAccountButton = cls.driver.find_element_by_accessibility_id('DeleteButton')
+        deleteAccountButton.click()
+        okButton = cls.driver.find_element_by_accessibility_id('OK')
+        okButton.click()
+        time.sleep(3)
```

Comment:
We need to make sure state changed, any chance  we can listen for changes on some item?